server/files/stylesheets/docs.css

```diff
@@ -1616,6 +1616,9 @@ code:hover .django .tag .keyword {
   color: #892A6F;
   font-weight: normal;
 }
+.item:hover svg g{
```

Comment:
This selector isn't specific enough, as it would affect all `.item` (menu item, dropdown item, etc).
The desired change should only affect the footer links so we should make the seector more specific
```css
.footer .ui.link.list > .item:hover svg g{
```

There also is a dedicated section inside docs.css for the footer adjustments (Line 29+) Please do your changes there instead of inside the code tag section for a better overview.

Reply:
Okay let me make changes.